stable/database/files/nuosm

```diff
@@ -467,9 +467,15 @@ function loadFromSnapshot() {
   fi
 
   if [ "$recreate_archives" == "true" ]; then
+    # Create restored.txt to signal that snapshot preparation is complete. This
+    # is needed in the absence of info.json, which is not created for the
+    # restored archive object until later.
+    echo "$BACKUP_ID" > "${DB_DIR}/restored.txt"
```

Comment:
Micro-optimization: why not `mv "${DB_DIR}/info.json" "${DB_DIR}/restored.txt"`? It closes the (extremely unlikely) window if the SM crashes between those two commands and we are left with an Archive that is not registered with the admin and does not want to try.

Reply:
What is the window that you're trying to close? The presence of `backup.txt` should not inhibit creation of the new archive object.

Reply:
Never mind, I missed a `else` branch.